src/Person.jsx

```diff
@@ -0,0 +1,36 @@
+import './App.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried,
+    partnerName = 'I am not married' } = person;
```

Comment:
The partnerName should be a name. You can create a separate variable for a partner and assign a value here 
```suggestion
  const { 
    name, 
    age, 
    sex, 
    isMarried,
    partnerName
  } = person;
```